docs/crd.md

```diff
@@ -7,399 +7,183 @@ whether the job spec is specified. Similarly to other kinds of Kubernetes resour
-      - **jobManagerService**: The status of the JobManager service.
-        - **name**: The resource name of the JobManager service.
-        - **state**: The state of the JobManager service.
-        - **nodePort** (optional): The node port, present when `accessScope` is `NodePort`.
-      - **jobManagerIngress**: The status of the JobManager ingress.
-        - **name**: The resource name of the JobManager ingress.
-        - **state**: The state of the JobManager ingress.
-        - **urls**: The generated URLs for JobManager.
-      - **taskManagerStatefulSet**: The status of the TaskManager StatefulSet.
-        - **name**: The resource name of the TaskManager StatefulSet.
-        - **state**: The state of the TaskManager StatefulSet.
-      - **job**: The status of the job.
-        - **id**: The ID of the Flink job.
-        - **name**: The Name of the Flink job.
-        - **completionTime**: Job completion time. Present when job is terminated regardless of its state.
-        - **state**: The state of the job.
-        - **fromSavepoint**: The actual savepoint from which this job started.
-          In case of restart, it might be different from the savepoint in the
-          job spec.
-        - **savepointGeneration**: The generation of the savepoint in `savepointsDir` taken by the operator. The value
-          starts from 0 when there is no savepoint and increases by 1 for each
-          successful savepoint.
-        - **savepointLocation**: Last savepoint location.
-        - **lastSavepointTriggerID**: Last savepoint trigger ID.
-        - **lastSavepointTime**: Last successful or failed savepoint operation timestamp.
-        - **restartCount**: The number of restarts.
-        - **failureReasons**: Reasons for the job failure. Present if job state is Failure.
-    - **control**: The status of control requested by user.
-      - **name**: Requested control name.
-      - **details**: Control details.
-      - **state**: Control state.
-      - **message**: Control message.
-      - **updateTime**: The updated time of control status.
-    - **savepoint**: The status of savepoint progress.
-      - **jobID**: The ID of the Flink job.
-      - **triggerID**: Savepoint trigger ID.
-      - **triggerTime**: Savepoint triggered time.
-      - **triggerReason**: Savepoint triggered reason.
-      - **state**: Savepoint state.
-      - **message**: Savepoint message.
-    - **currentRevision**: CurrentRevision indicates the version of FlinkCluster.
-    - **nextRevision**: NextRevision indicates the version of FlinkCluster updating.
-    - **collisionCount**: collisionCount is the count of hash collisions for the FlinkCluster.
-      The controller uses this field as a collision avoidance mechanism
-      when it needs to create the name for the newest ControllerRevision.
-    - **lastUpdateTime**: Last update timestamp of this status.
+| Key | Type | Optional | Description
```

Comment:
```suggestion
| Key | Type | Required | Description
```

and then we could only mark the required fields. wdyt?

Reply:
good! XOXOXOXXX make us confuse

Reply:
https://github.com/moonsub-kim/flink-on-k8s-operator/blob/crd/docs/crd.md